docs/modules/WebDriver.md

```diff
@@ -24,13 +24,13 @@ To run Selenium Server you will need Java and Chrome or Firefox browser installe
 
 ### PhantomJS
 
-PhantomJS is a headless alternative to Selenium Server that implements
+PhantomJS is a [headless browser](https://en.wikipedia.org/wiki/Headless_browser) alternative to Selenium Server that implements
```

Comment:
OK, see https://github.com/ThomasLandauer/Codeception/commit/77fe6ef59df29a2293e7e589ffe62de8b66b94f0

* Is it really a good idea to have hundreds of lines of documentation inside the source code?
* I'm losing track of my commits ;-) Could you merge them and/or invoke the markdown-conversion manually?